docs/img-src/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
ok, I will try a few more ways and link them here so we can discuss it on a visual basis